music/genre.rb

```diff
@@ -1,15 +1,15 @@
+require '../../item'
```

Comment:
![cantfile](https://github.com/torobucci/catalog-of-my-things/assets/26763433/82a8da4d-ec25-4f9d-95da-2318b98b57b9)

Reply:
- [ ] Hello @Odongo006, It is a requirement to "add unit tests for all implemented methods" So please make sure you implement them for all methods